content/operator/ltg/index.de.md

```diff
@@ -0,0 +1,94 @@
+---
+draft: false
+title: "LTG-Link"
+country:
+  - "lithuania"
+  - "latvia"
+operator: "ltg"
+---
+
+Die LTG ist die staatliche Eisenbahngesellschaft der Republik Litauen und betreibt mit ihrer Tochtergesellschaft LTG-Link alle Verbindungen im Personenverkehr auf dem litauischen Schienennetz sowie ins Ausland.
+
+## Zusammenfassung
+
+- FIP 50 und FIP Freifahrtscheine werden akzeptiert.
+- Reservierungspflicht in allen Zügen.
+- Bei Nutzung des FIP Freifahrtscheins muss vor jeder Fahrt ein kostenloses Ticket erworben werden.
+
+## Gültigkeit FIP Tickets
+
+FIP Freifahrtschein: ✅ \
+FIP Freifahrt Angehörige: ⛔ \
+FIP 50 Tickets: ✅
+
+{{% highlight important %}}
+Bei allen Fahrten mit einem FIP Freifahrtschein muss vor jeder Fahrt ein kostenloses Ticket bzw. Reservierung erworben werden ([siehe Zugkategorien und Reservierungen](#zugkategorien-und-reservierungen)).
+{{% /highlight %}}
+
+FIP Freifahrtscheine und FIP 50 Tickets sind auf Verbindungen der LTG-Link gültig. Bei grenzüberschreitenden Fahrten nach/von Polen müssen entweder ein durchgängiges FIP 50 Ticket oder FIP Freifahrtscheine beider Länder vorhanden sein; für Fahrten von/nach Lettland wird nur ein litauischer FIP Freifahrtschein bzw. ein FIP 50 Ticket verlangt, da Lettland kein FIP-Mitglied ist.
+
+{{% highlight tip %}}
+FIP 50 Tickets und Freifahrtscheine von LTG-Link, gelten auch in Zügen von LTG-Link auf der Strecke Vilnius – Riga. [^1]
+{{% /highlight %}}
+
+## Zugkategorien und Reservierungen
+
+In Litauen wird nicht zwischen verschiedenen Zugkategorien unterschieden.
+
+{{% highlight important %}}
+LTG-Link verlangt in Fernverkehrszügen eine verpflichtende kostenlose Sitzplatzreservierung (eine Auflistung der Verbindungen ist auf [LTG-Link Website](https://ltglink.lt/en/passengers-with-interrail-eurail-passes) zu finden). Mit FIP Freifahrtschein muss jedoch auch in Zügen ohne Reservierungspflicht ein kostenloses Ticket (_0er-Fahrschein_) vor der Fahrt erworben werden.
+
+Züge mit Reservierungspflicht sind teilweise schon viele Tage im Voraus ausgebucht. In der Hochsaison und an Feiertagen zwischen Vilnius und Klaipeda bzw. Riga sind Züge teilweise 2-3 Wochen zuvor ausgebucht.
+
+Buchungsbeginn ist bei Inlandsfahrkarten 30 Tage vor Abfahrt und bei Auslandsfahrten 60 Tage vor Abfahrt.
+
+Somit ist vor **jeder** Fahrt mit FIP Freifahrtschein ein kostenloses Ticket (0er-Fahrschein) mit/ohne Pflichtreservierung oder ein FIP 50 Ticket zu erwerben, das ggf. eine kostenlose Reservierung enthält. (siehe [Ticket- und Reservierungskauf](#ticket--und-reservierungskauf)).
+{{% /highlight %}}
+
+FIP Vergünstigungen der 1. Klasse gelten in allen Klassen. FIP Vergünstigungen der 2. Klasse gelten in der 2. und 3. Klasse.
+
+Fahrgäste der 1. Klasse erhalten in Fernverkehrszügen kostenlose Getränke und einen kleinen Imbiss.
```

Comment:
Das ist abhängig von der Strecke, siehe ganz unten hier: https://ltglink.lt/en/benefits-of-first-class

> On the Vilnius-Klaipeda-Vilnius, Vilnius-Riga-Vilnius and Vilnius-Turmantas-Vilnius routes:
>    If your journey takes up to 1 hour, we will treat you to one complimentary drink of your choice: coffee, tea or a bottle of water.
>    If your journey time is more than 1 hour, we will treat you to a drink of your choice (coffee/tea), a bottle of water and a chocolate brownie.
> 
> On the Vilnius-Kaunas-Vilnius and Vilnius-Mockava-Vilnius routes:
>
>    If you are travelling on the Vilnius-Kaunas-Vilnius or Vilnius-Mockava-Vilnius routes, we will treat you to a bottle of mineral water!